recipes/hazelcastcxx/all/test_package/conanfile.py

```diff
@@ -0,0 +1,22 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class TestPackageHazelcastCxx(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package_multi"
+    requires = "hazelcastcxx/[>4.0.0]"
+
+    def configure(self):
+        if self.settings.compiler.cppstd:
+            tools.check_min_cppstd(self, 11)
```

Comment:
You should be settings the `CXX_STandard` in the CMake  (you can find examples in the repo)

Reply:
Done.